### ISSUE TITLE
laststatus=0 : winheight changes after horizontal split

### ISSUE THREAD

**Issue by ags-rnd** [neovim/neovim#5354]
- `nvim --version`: NVIM 0.1.5
- Vim (version: 7.4) behaves differently? Yes
- Operating system/version: arch 4.7.1-1-ARCH
- Terminal name/version: st 0.6
- `$TERM`: st-256color
### Actual behaviour

When `laststatus == 0` and `splitbelow` is set with the opening of every horizontal split nvim decreases the total winheight decreases by one.
Basically, with every toggle of an horizontal window, vim eats a line, which is very annoying.
### Expected behaviour

Initially we have a window, with height, say `h_prev`, if we do an horizontal split with `:split` and then close it, the new height, say `h_new`, is such that, `h_new = h_prev - 1`
### Steps to reproduce using `nvim -u NORC`

```
nvim -u NORC             # fire up nvim
:set splitbelow
:set laststatus=0
:echo winheight ('%')    # annotate this value: 'h_prev'
:split
:q
:echo winheight ('%')    # annotate this value: 'h_new'
```

Observe that `h_new = h_prev - 1`

**Comment by ags-rnd** [comment]
Wasn't considering `splitbelow` option

**Comment by justinmk** [comment]
Seems to get adjusted when user enters command-line (`:`).

**Comment by danlrobertson** [comment]
I'll take a crack at this

**Comment by justinmk** [comment]
Closed by #5705
